core/src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -1813,512 +1273,6 @@ object SampleWdl {
-        |    memory: "5 TY"
-        |  }
-        |}
-        |
-        |workflow great_googly_moogly {
-        |  call messed_up_memory
-        |}
-      """.stripMargin
-
-    val rawInputs = Map.empty[String, Any]
-  }
-
-  object WorkflowWithMessedUpLocalDisk extends SampleWdl {
-    override def wdlSource(runtime: String = "") =
-      """
-        |task messed_up_disk {
-        |  command {
-        |      echo "YO"
-        |  }
-        |  runtime {
-        |    docker: "ubuntu:latest"
-        |    disks: "bad_mount 123 LOCAL"
-        |  }
-        |}
-        |
-        |workflow great_googly_moogly {
-        |  call messed_up_disk
-        |}
-      """.stripMargin
-
-    val rawInputs = Map.empty[String, Any]
-  }
-
-  object WorkflowWithMessedUpDiskSize extends SampleWdl {
-    override def wdlSource(runtime: String = "") =
-      """
-        |task messed_up_disk {
-        |  command {
-        |      echo "YO"
-        |  }
-        |  runtime {
-        |    docker: "ubuntu:latest"
-        |    disks: "/mnt 123.0 SSD"
-        |  }
-        |}
-        |
-        |workflow great_googly_moogly {
-        |  call messed_up_disk
-        |}
-      """.stripMargin
-
-    val rawInputs = Map.empty[String, Any]
-  }
-
-  object WorkflowWithMessedUpDiskType extends SampleWdl {
-    override def wdlSource(runtime: String = "") =
-      """
-        |task messed_up_disk {
-        |  command {
-        |    echo "YO"
-        |  }
-        |  runtime {
-        |    docker: "ubuntu:latest"
-        |    disks: "/mnt 123 SDD"
-        |  }
-        |}
-        |
-        |workflow great_googly_moogly {
-        |  call messed_up_disk
-        |}
-      """.stripMargin
-
-    val rawInputs = Map.empty[String, Any]
-  }
-
-  object WorkflowWithRuntimeAttributeExpressions extends SampleWdl {
-    override def wdlSource(runtime: String = "") =
-      """
-        |task test {
-        |  String tag
-        |  Int disk_gb
-        |  Int memory_gb
-        |
-        |  command { ps }
-        |  runtime {
-        |    docker: "ubuntu:" + tag
-        |    disks: "/mnt " + disk_gb + " SSD"
-        |    memory: memory_gb + " GB"
-        |  }
-        |}
-        |
-        |workflow w {call test}
-      """.stripMargin
-
-    val rawInputs = Map(
-      "w.test.tag" -> "latest",
-      "w.test.disk_gb" -> 9,
-      "w.test.memory_gb" -> 7
-    )
-  }
-
-  object WorkflowWithRuntimeAttributeExpressions2 extends SampleWdl {
-    override def wdlSource(runtime: String = "") =
-      """
-        |task x {
-        |  Int a
-        |  Int b = a * 1000
-        |  String c = b + "MB"
-        |  command { ps }
-        |  runtime {
-        |    docker: "ubuntu:latest"
-        |    memory: c
-        |  }
-        |}
-        |
-        |workflow w {call x}
-      """.stripMargin
-
-    val rawInputs = Map(
-      "w.x.a" -> 5
-    )
-  }
-
-  object WorkflowWithFourPreemptibleRetries extends SampleWdl {
```

Comment:
TOL: I don't think we do preemptible testing at all in Centaur-- for very understandable reasons. Kind of along the same lines as testing restarts in Centaur, it will require extra magic.